packages/web-api-scan-runner/src/report-generator/axe-result-to-html-converter.ts

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { AxeReportParameters, ReporterFactory } from 'accessibility-insights-report';
+import { AxeResults } from 'axe-core';
+import { inject, injectable } from 'inversify';
+import { ReportFormat } from 'storage-documents';
+import { iocTypeNames } from '../ioc-types';
+import { AxeResultConverter, ReportGenerationParams } from './axe-result-converter';
+
+@injectable()
+export class AxeResultToHtmlConverter extends AxeResultConverter {
+    public readonly reportType: ReportFormat = 'html';
+
+    constructor(@inject(iocTypeNames.ReporterFactory) private readonly reporterFactoryFunc: ReporterFactory) {
+        super();
+    }
+
+    public convert(results: AxeResults, params: ReportGenerationParams): string {
+        const reporter = this.reporterFactoryFunc();
+
+        const htmlReportParams: AxeReportParameters = {
+            results: results,
+            description: 'Automated report',
```

Comment:
Can it be more descriptive? What is PM recommendation here?

Reply:
Addie suggested adding scan url and time. I've updated this to include those.